src/Inventory.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Inventory.h"
+#include "Config.h"
+Inventory::Inventory() {
+	m_windows.resize( Config::Inventory::number_of_windows );
+	for ( int i = 0; i < Config::Inventory::number_of_windows; ++i ) {
+		m_windows[ i ].setSize( Config::Inventory::window_size );
+		m_windows[ i ].setFillColor( Config::Inventory::fill_color );
+		m_windows[ i ].setOutlineColor( Config::Inventory::outline_color );
+		m_windows[ i ].setOutlineThickness( Config::Inventory::thickness );
+		//m_windows[ i ].setPosition( { i * 42.f - Config::Window::width / 2 , 10.f } );
+	}
+	current = &m_windows[ current_pos ];
+	current->setOutlineColor( sf::Color::Red );
+}
+const std::vector< sf::RectangleShape >& Inventory::inventory() const {
+	return m_windows;
+}
+
+void Inventory::set_position( const sf::Vector2f& pos ) {
+	for ( int i = 0; i < Config::Inventory::number_of_windows; ++i ) {
+		m_windows[ i ].setPosition( { pos.x + i * 42 - Config::Window::width / 4 , 10.f } );
+	}
+}
+
+void Inventory::move( int moved_by ) {
+	current_pos += moved_by;
+	if ( current_pos == Config::Inventory::number_of_windows ) { current_pos = 0; }

```

Comment:
Please put block of code into a new line. You put it into a new line elsewhere, so let's keep the consistency.